app/controllers/UkimsNumberController.scala

```diff
@@ -104,4 +107,45 @@ class UkimsNumberController @Inject() (
             } yield Redirect(navigator.nextPage(UkimsNumberUpdatePage, mode, updatedAnswers))
         )
   }
+
+  def onPageLoadCreateNew(mode: Mode): Action[AnyContent] =
+    (identify andThen getData andThen requireData) { implicit request =>
+      val preparedForm = request.userAnswers.get(UkimsNumberPage) match {
```

Comment:
I am wondering if we should create a new page, something like `NewUkimsNumberPage`?

Reply:
Hmmmn, I debated that, but since this new route and view is supposed to update the existing one (`UkimsNumberPage`), it makes sense to populate the form with the existing value.

Reply:
I think I agree with Yohan, since we have two views in one controller - I would rather split them out, so we have a controller per view. I know they are both updating the UkimsNumberPage, but they're called in two separate scenarios. 

This could also be a good place to start moving stuff into logical packages such as the below, but I'll leave that for you to choose if you want
controllers > ukims > ukimsnumbercontroller
controllers > ukims > newukimsnumbercontroller

Reply:
This is another conflict I had with the controller, the ticket says:
- Update session data with user answers
- Use same controller

I don't know where that came from, but I guess I have to update the ticket and create a new controller

Reply:
I have raised a concern with Amaka. There is a ticket for the nav but there is not ticket for the new/ukims/cya page which is in the nav and the mural board
![Screenshot from 2024-11-05 13-57-13](https://github.com/user-attachments/assets/bd6ce5df-743d-4b5c-b9d8-b12903abb50b)